scripts/lynx/upgrade_coordinator.py

```diff
@@ -18,12 +19,22 @@ def main():
     deployer = Deployer.from_yaml(filepath=CONSTRUCTOR_PARAMS_FILEPATH, verify=VERIFY)
     instances = contracts_from_registry(filepath=ARTIFACTS_DIR / "lynx.json", chain_id=80002)
 
-    coordinator = deployer.upgrade(
-        project.Coordinator, instances[project.Coordinator.contract_type.name].address
+    implementation = deployer.deploy(project.Coordinator)
+    # encoded_initializer_function = implementation.initializeNumberOfRituals.encode_input()
+    encoded_initializer_function = b""
```

Comment:
Do we no longer need the commented line above?

Reply:
it's needed as one time thing, I didn't want to remove it to have it as example for other network including mainnet

Reply:
Ah ok. Makes sense.

Reply:
Made a suggestion for a comment to add about that - you can edit/adjust as you see fit.